modules/cco-ccoctl-gcp-image-registry.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc
+
+:_content-type: PROCEDURE
+[id="cco-ccoctl-gcp-image-registry_{context}"]
+= Configuring the image registry to use long-lived credentials
+
+In {product-title} 4.10.8, support for using GCP Workload Identity for the image registry is removed from {product-title} 4.10, and you must configure the image registry to use long-lived credentials. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.
```

Comment:
I might suggest rewording this to (as it was a bit confusing and it would be nice if we could give a reason why):

In {product-title} 4.10.8, image registry support is removed when you use GCP Workload Identity [due to ???]. To use the image registry in {product-title} 4.10.8, you must configure the image registry to use long-lived credentials instead. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.

Reply:
Thanks, Ju! How do you feel about using this bit from the rel notes?

> due to the discovery of [an adverse impact to the image registry](https://bugzilla.redhat.com/show_bug.cgi?id=2069807).